arpack-ng-config.cmake.in

```diff
@@ -17,9 +17,43 @@ set(exec_prefix "@exec_prefix@")
 set(libdir "@libdir@")
 set(includedir "@includedir@")
 
+include(CMakeFindDependencyMacro)
```

Comment:
If `find_dependency` are done, sounds like one must kill `find_package` in CMakeLists.txt? https://github.com/opencollab/arpack-ng/blob/645db9e6472283408563e3e2dc99c5deb8272c5d/CMakeLists.txt#L222

Reply:
No, find_package() is still required in CMakeLists.txt to determine how the library is built and to build the executables and tests themselves. find_dependency() is only used in the installed copy of arpack and only when the library is searched for by the user.